src/main/java/com/jfb/ticket_app/service/TicketService.java

```diff
@@ -1,46 +1,56 @@
 package com.jfb.ticket_app.service;
 
 import com.jfb.ticket_app.model.ticket.Ticket;
-import com.jfb.ticket_app.model.ticket.enums.TicketType;
-import com.jfb.ticket_app.repository.dao.TicketDAO;
-import com.jfb.ticket_app.util.interfaces.Identifiable;
 
+import com.jfb.ticket_app.repository.TicketRepository;
 import java.util.List;
 
+import java.util.Optional;
+import java.util.UUID;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
-public class TicketService implements Identifiable {
+public class TicketService {
 
-  private final String CLASS_ID = generateId();
-  private final TicketDAO ticketDAO;
+  private TicketRepository ticketRepository;
 
   @Autowired
-  public TicketService(TicketDAO ticketDAO) {
-    this.ticketDAO = ticketDAO;
+  public TicketService(TicketRepository ticketRepository) {
+    this.ticketRepository = ticketRepository;
   }
 
   public void saveTicket(Ticket ticket) {
-    ticketDAO.saveTicket(ticket);
+    ticketRepository.save(ticket);
   }
 
-  public Ticket getTicketById(String id) {
-    return ticketDAO.getTicketById(id);
+  public Ticket getTicketById(UUID id) {
+    Optional<Ticket> ticket = ticketRepository.findById(id);
+    return ticket.orElse(null);
```

Comment:
use `.orElseThrow(...)` instead `orElse(null)` and throw your custom or some specific exception.